src/agents/workflows/review_logic.py

```
@@ -0,0 +1,237 @@
+
+def _serialize_model_obj(obj):
+    try:
+        return obj.model_dump(exclude_none=True) if hasattr(obj, "model_dump") else obj.dict(exclude_none=True)
+    except Exception:
+        return str(obj)
+
+
+def _extract_json(text: str) -> Optional[Dict[str, Any]]:
+    try:
+        return json.loads(text)
+    except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)